dev/src/reference/query-util.ts

```diff
@@ -254,10 +258,18 @@ export class QueryUtil<
         // catch below.
         let request = query.toProto(transactionOrReadTime, explainOptions);
 
+        let isRetryRequestWithCursor = false;
         let streamActive: Deferred<boolean>;
         do {
           streamActive = new Deferred<boolean>();
           const methodName = 'runQuery';
+
+          this._firestore._traceUtil.currentSpan().addEvent(methodName, {
+            [ATTRIBUTE_KEY_IS_TRANSACTIONAL]:
+              request.transaction !== null && request.transaction !== undefined,
```

Comment:
can this be replaced by `!!request.transaction`?

Reply:
yes, good one. Done.